raiden/network/transport/matrix/transport.py

```diff
@@ -738,19 +738,27 @@ def _initialize_first_sync(self) -> None:
         assert self._client.sync_thread is None, msg
         assert self._client.message_worker is None, msg
 
+        # search for broadcast room ids via public rooms list
+        broadcast_room_ids = self._client.public_room_ids(make_room_alias(self.chain_id, ""))
+
+        # TODO: Uncomment once the server updated their public room list
```

Comment:
When does the server do this?

Reply:
Swapping `initialize_broadcast_rooms` and `initialize_first_sync` removes the necessity of the publicrooms call. check out the latest push